engine/src/main/scala/cromwell/server/CromwellRootActor.scala

```diff
@@ -27,13 +25,14 @@ import cromwell.engine.workflow.WorkflowManagerActor.AbortAllWorkflowsCommand
 import cromwell.engine.workflow.lifecycle.execution.callcaching.{CallCache, CallCacheReadActor, CallCacheWriteActor}
 import cromwell.engine.workflow.lifecycle.finalization.CopyWorkflowLogsActor
 import cromwell.engine.workflow.tokens.{DynamicRateLimiter, JobExecutionTokenDispenserActor}
-import cromwell.engine.workflow.workflowstore.{SqlWorkflowStore, WorkflowStore, WorkflowStoreActor}
+import cromwell.engine.workflow.workflowstore.{SqlWorkflowStore, WorkflowHeartbeatConfig, WorkflowStore, WorkflowStoreActor}
 import cromwell.jobstore.{JobStore, JobStoreActor, SqlJobStore}
 import cromwell.services.{EngineServicesStore, ServiceRegistryActor}
 import cromwell.subworkflowstore.{SqlSubWorkflowStore, SubWorkflowStoreActor}
 import cromwell.util.GracefulShutdownHelper
 import cromwell.util.GracefulShutdownHelper.ShutdownCommand
 import net.ceedubs.ficus.Ficus._
+import sext._
```

Comment:
What is this being used for?

Reply:
Line 63. The output looks like:

```
[INFO] [04/07/2018 20:47:33.646] [cromwell-system-akka.dispatchers.engine-dispatcher-5] [akka://cromwell-system/user/cromwell-service] Workflow heartbeat configuation:
- cromwellId:
| cromid-286ccb0
- ttl:
| 10 minutes
- sendsPerTtl:
| 60
- writeBatchSize:
| 100
- writeBatchesPerTtl:
| 60
- writeThreshold:
| 100
```

Reply:
Though actually with an `Encoder[FiniteDuration]` Circe can derive an `Encoder[WorkflowHeartbeatConfig]` that produces this which is probably just as good:

```
{
  "cromwellId" : "cromid-80eb68c",
  "ttl" : "10 minutes",
  "sendsPerTtl" : 3,
  "writeBatchesPerTtl" : 3,
  "writeBatchSize" : 100,
  "writeThreshold" : 100
}
```